tests/runtime_neutral/test_discoverable_wrapper_host_visibility.py

```
@@ -107,6 +108,46 @@ def test_shell_check_fails_when_a_wrapper_entry_is_missing(self) -> None:
             self.assertNotEqual(0, result.returncode)
             self.assertIn("[FAIL] host-visible discoverable entries", result.stdout)
 
+    def test_shell_check_fails_when_public_wrapper_skill_is_missing_even_if_hidden_bundle_exists(self) -> None:
+        self._require_bash()
+        with tempfile.TemporaryDirectory() as tempdir:
+            target_root = Path(tempdir) / "codex-root"
+            subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)